tools/msPurity/macros.xml

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0"?>
 <macros>
-    <token name="@TOOL_VERSION@">1.11.4</token>
-    <token name="@GALAXY_TOOL_VERSION@">0.2.7</token>
+    <token name="@TOOL_VERSION@">1.12.0</token>
+    <token name="@GALAXY_TOOL_VERSION@">0</token>
 
     <xml name="requirements">
         <requirements>
            <requirement type="package" version="@TOOL_VERSION@" >bioconductor-mspurity</requirement>
+           <requirement type="package" version="1.42.0" >bioconductor-camera</requirement>
+           <requirement type="package" version="3.8.0" >bioconductor-xcms</requirement>
+           <requirement type="package" version="1.14.0" >bioconductor-mspuritydata</requirement>
+           <requirement type="package" >r-optparse</requirement>
```

Comment:
please use a version number ....

Reply:
Updated with version numbers for all packages